src/components/Header.vue

```diff
@@ -4,11 +4,17 @@
       <h3>🌌 Demo dApp</h3>
     </div>
     <div class="address">
-      <span v-if="acct">Account: {{ acct.substring(0, 16) }}...</span>
-      <span v-if="userName">@{{ userName }}</span>
-      <span v-else>
+      <!-- <span v-if="acct">Account: {{ acct.substring(0, 16) }}...</span> -->
+      <div v-if="userName">
+        <div>@{{ userName }}</div>
+        <div v-if="balance">{{ balance }}</div>
+        <div v-else>
+          <button class="btn">CLAIM 500 TOKENS</button>
```

Comment:
current contract logic is that when a user signs up for the first time, they are automatically sent 500 tokens